fastcli/fastcli.py

```diff
@@ -139,8 +139,16 @@ def cli() -> None:
         help="increase verbosity (may repeat up to -vvv)",
     )
     namespace = parser.parse_args()
-    args = {k: v for k, v in vars(namespace).items() if v}
-    speed = run(**args)
+    args = {k: v for k, v in vars(namespace).items()}
```

Comment:
The `if v` is required to exclude kwargs that default to falsey values like `None`. Read more about this pattern at this reddit thread I started a few years ago: https://www.reddit.com/r/learnpython/comments/4jv8xo/argparse_using_parser_function_while_not/